src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java

```diff
@@ -27,39 +27,39 @@
   private static final String ERROR_MESSAGE =
       "We don't recognize these credentials. Try again or register a new account.";
 
-  @Test(groups = "MercuryLoginTest_002")
+  @Test(groups = {"MercuryLoginTest_002", "LoginTests_001"})
   public void MercuryLoginTest_002_userCanNotLogInWithWrongPassword() {
     LoginPageObject loginPageObject = new LoginPageObject(driver).get();
     loginPageObject.logUserIn(Configuration.getCredentials().userName10, "thisIsWrongPassword");
 
     Assertion.assertEquals(loginPageObject.getErrorMessage(), ERROR_MESSAGE);
   }
 
-  @Test(groups = "MercuryLoginTest_003")
+  @Test(groups = {"MercuryLoginTest_003", "LoginTests_001"})
   public void MercuryLoginTest_003_invalidUserCanNotLogIn() {
     LoginPageObject loginPageObject = new LoginPageObject(driver).get();
     loginPageObject.logUserIn("notExistingUserName", Configuration.getCredentials().password10);
 
     Assertion.assertEquals(loginPageObject.getErrorMessage(), ERROR_MESSAGE);
   }
 
-  @Test(groups = "MercuryLoginTest_004")
+  @Test(groups = {"MercuryLoginTest_004", "LoginTests_001"})
   public void MercuryLoginTest_004_notPossibleToLogInWhenUsernameFieldBlank() {
     LoginPageObject loginPageObject = new LoginPageObject(driver).get();
     loginPageObject.logUserIn("", Configuration.getCredentials().password10);
 
     Assertion.assertTrue(loginPageObject.isSubmitButtonDisabled());
   }
 
-  @Test(groups = "MercuryLoginTest_005")
+  @Test(groups = {"MercuryLoginTest_005", "LoginTests_001"})
   public void MercuryLoginTest_005_notPossibleToLogInWhenPasswordFieldBlank() {
     LoginPageObject loginPageObject = new LoginPageObject(driver).get();
     loginPageObject.logUserIn(Configuration.getCredentials().userName10, "");
 
     Assertion.assertTrue(loginPageObject.isSubmitButtonDisabled());
   }
 
-  @Test(groups = "MercuryLoginTest_006")
+  @Test(groups = {"MercuryLoginTest_006", "LoginTests_001"})
   public void MercuryLoginTest_006_closeButtonWorksAndRedirectsProperly() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)